libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -115,6 +115,25 @@ class AP_AHRS
         return _flags.fly_forward;
     }
 
+    void set_done_takeoff(bool b) {
+        if (b && !_flags.done_takeoff) {
+            _last_takeoff_ms = AP_HAL::millis();
+        }
+        _flags.done_takeoff = b;
+    }
+
+    bool get_done_takeoff(void) const {
+        return _flags.done_takeoff;
+    }
+
+    // return time in milliseconds since last takeoff
+    uint32_t time_since_takeoff_ms(void) const {
```

Comment:
We need to explicitly state in the interface the expected behaviour when the vehicle is on the ground. The comment implies that it continues to increment.